### PR TITLE
[ZT] Technical limitations -> account limits

### PR DESCRIPTION
**Pull request by ranbel**

### CODE REVIEW

File: content/cloudflare-one/account-limits.md [hunk 1]
```diff
@@ -1,36 +1,49 @@
...
+## Access
 
 | Feature                                | Limit |
 | -------------------------------------- | ----- |
 | Applications count                     | 500   |
 | Audit Logpush jobs                     | 5     |
-| DNS Logpush jobs                       | 5     |
 | Email addresses per rule               | 1,000  |
 | Group count                            | 300   |
 | Group size                             | 1,000  |
```
Comment (ranbel) [inline content/cloudflare-one/account-limits.md:19]: @kennyj42 What is group size? How is it different from "Rules count per group"?

File: content/cloudflare-one/account-limits.md [hunk 2]
```diff
@@ -1,36 +1,49 @@
...
-| HTTP Logpush jobs                      | 5     |
 | IP addresses per rule                  | 1,000  |
```
Comment (ranbel) [inline content/cloudflare-one/account-limits.md:20]: @Encore-Encore Does "IP addresses per rule" apply to Gateway expressions?

File: content/cloudflare-one/account-limits.md [hunk 3]
```diff
@@ -1,36 +1,49 @@
...
-| Lists                                  | 100   |
-| Locations                              | 250   |
 | mTLS root certificates                 | 50    |
-| Number of HTTP policies per account    | 500   |
-| Number of DNS policies per account     | 500   |
-| Number of Network policies per account | 500   |
 | Service tokens count                   | 50    |
 | IdP count                              | 50    |
 | Rules count per application            | 1,000  |
 | Rules count per group                  | 1,000  |
```
Comment (ranbel) [inline content/cloudflare-one/account-limits.md:25]: @Encore-Encore What's the limit for # of rules/expressions per Gateway policy?

File: content/cloudflare-one/account-limits.md (hunk 1 above)
Comment (kennyj42) [reply content/cloudflare-one/account-limits.md:19]: I believe it's the total number of records allowed in any given group. E.g. you can only have 1k IP addresses in a group. @GreenStage is that correct?

**Review by mmalden — APPROVED** [review]

**Review by Princeobinna — COMMENTED** [review]
I want my assets

**Pull request merged**